packages/core/core/src/utils/nuxt/_proxyUtils.ts

```diff
@@ -25,25 +25,16 @@ export const createProxiedApi = ({ givenApi, client, tag }: CreateProxiedApiPara
 export const getCookies = (context: NuxtContext) => context?.req?.headers?.cookie ?? '';
 
 export const getIntegrationConfig = (context: NuxtContext, configuration: any) => {
+  const baseURL = context?.$config?.middlewareUrl;
   const cookie = getCookies(context);
 
-  if (context?.$config?.middlewareUrl) {
-    const { middlewareUrl } = context.$config;
-    return merge({
-      axios: {
-        baseURL: middlewareUrl,
-        headers: {
-          ...(cookie ? { cookie } : {})
-        }
-      }
-    }, configuration);
-  }
-
   return merge({
     axios: {
+      baseURL: /\/api\//gi.test(baseURL) ? baseURL : `${baseURL}/api/`,
```

Comment:
This is enforcing to always use `/api/` at the end of url, which is not desired when using external middleware like this one: https://docs.vuestorefront.io/v2/architecture/server-middleware.html#separating-server-middleware-from-nuxt-js

Reply:
Let's say it's a global standard of naming convention.